components/InterestZoneAdd/InterestZoneAdd.tsx

```diff
@@ -0,0 +1,48 @@
+import DetailsForm from '../InterestZoneView/DetailsForm';
+import styles from '../InterestZoneView/InterestZoneView.module.css';
+
+import Box from '@mui/material/Box';
+import Modal from '@mui/material/Modal';
+import { useCallback, useState } from 'react';
+import Button from '@mui/material/Button';
+
+interface Props{
+    onClose: () => void,
+    isVisible: boolean,
+    /// must have long and lat
+}   
+
+const InterestZoneAdd = (props: Props) => {
+
+    const handleSave = () => {
+    };
+
+    const handleClose = useCallback(() => {
+        props.onClose();
+    }, []);
+
+    const handleZoneInputChange = () => {
+        
+    };
+
+    return (
+        <Modal
+          open={props.isVisible}
+          onClose={handleClose}
+          aria-labelledby="modal-modal-details"
+          aria-describedby="modal-modal-zone-details"
```

Comment:
Please omit `aria` related properties, these are linked to accessibility which is not a concern at the moment